gumbo-parser/fuzzer/build-ossfuzz.sh

```diff
@@ -0,0 +1,7 @@
+cd src
+make
+cd ../
+
+$CXX $CXXFLAGS -o $OUT/parse_fuzzer fuzzer/parse_fuzzer.cc src/libgumbo.a $LIB_FUZZING_ENGINE
+cp fuzzer/gumbo.dict $OUT/parse_fuzzer.dict
+cp fuzzer/gumbo_corpus.zip $OUT/parse_fuzzer_seed_corpus.zip
```

Comment:
Looks like the last line is missing a newline.

Reply:
fixed (i hope) thanks for taking a look @stevecheckoway !